spec/amber_router/route_set/matching/routes_with_variables_spec.cr

```diff
@@ -40,6 +40,15 @@ describe "routes with variables" do
 
   it "routes with constraints" do
     router = build do
+      # With symbol hash
+      add "/get/int/:page_num", :page_num, {:page_num => :integer}
+
+      # With symbol hash
+      add "/get/string/:page_name", :page_name, {:page_name => :ascii}
```

Comment:
Since strings can contain the full range of unicode, can the url `/get/string/:page_name`, be replaced with`/get/ascii/:page_name`?